nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/TestCompoundDocumentRest.java

```diff
@@ -118,12 +112,13 @@ public void testCompoundDocumentUpload() throws IOException {
             JsonNode node = mapper.readTree(response.getEntityInputStream());
             batchId = node.get("batchId").asText();
         }
+        Blob blob = getTestArchive();
+        var fileName = FilenameUtils.removeExtension(blob.getFilename());
```

Comment:
According to the documentation the `X-File-Name` header should have the file extension: https://doc.nuxeo.com/nxdoc/howto-upload-file-nuxeo-using-rest-api/#file-upload